.travis.yml

```diff
@@ -8,11 +8,14 @@ jobs:
       dist: xenial
     - name: “Python 2.7 on bionic”
       python: "2.7"
-    - name: “Python 3.8 make lint only”
+    - name: “Python 3.8 make lint and some pytests”
       python: "3.8"
       install: pip install flake8
       script:
         - make lint
+        - pip install -r requirements_test.txt
+        - pushd vendor/infogami && git pull origin master && popd
```

Comment:
these 2 lines should be in the `install` section.

I was going to investigate whether there is a better way to get the latest `infogami` using `git submodule sync` or similar, but that can probably be a separate PR since the current method uses the `pushd` and `git pull` method.

Reply:
Blocked by https://github.com/internetarchive/openlibrary/pull/2951